DataFormats/NanoAOD/interface/LeptonTimeLifeInfo.h

```diff
@@ -0,0 +1,73 @@
+#ifndef DataFormats_NanoAOD_LeptonTimeLifeInfo_h
+#define DataFormats_NanoAOD_LeptonTimeLifeInfo_h
+
+/**
+  \class    LeptonTimeLifeInfo
+  \brief    Structure to hold lepton life-time information
+
+  \author   Michal Bluj, NCBJ, Warsaw
+*/
+
+#include "DataFormats/GeometryCommonDetAlgo/interface/GlobalError.h"
+#include "DataFormats/GeometryCommonDetAlgo/interface/Measurement1D.h"
+#include "DataFormats/GeometryVector/interface/GlobalPoint.h"
+#include "DataFormats/GeometryVector/interface/GlobalVector.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+
+class LeptonTimeLifeInfo {
```

Comment:
this class might belong elsewhere than NANO data formats, please investigate

Reply:
Hmm, it is used only to produce nano tables and it is why I placed it here. It is a kind of analysis format, so maybe can go to PatCandidates or similar. But I have not strong feeling. Anyway, it is not expected to be a persistent data-format (as mentioned in other comment).

Reply:
FWIW I do agree this looks better fitting somewhere else.

>  Anyway, it is not expected to be a persistent data-format (as mentioned in other comment).

the point is that I might want to be able to use this somewhere else in cmssw (e.g. in DQM or Alignment / Calibration) and don't want to drag in a dependence on `DataFormats/NanoAOD`.

Reply:
I tend to agree. Do you have proposal where it will fit well?

Reply:
>  Do you have proposal where it will fit well?

well, despite it's called `Lepton`-something I don't see anything super lepton-specific. I had in mind `DataFormats/VertexReco` (or `DataFormats/VertexReco`), perhaps after moving `Lepton` to `Track`.

Reply:
OK, fine with me. 
I have not strong objections against `Lepton` -> `Track` however tracks usually have not SV, but of course a track can originate from a SV. Generally, if you have a particular use-case in mind the structure can be modified to better fit this use-case as currently it is tailored to what is needed in this particular development.

Reply:
> you have a particular use-case in mind the structure can be modified to better fit this use-case.

the class layout looks good to me, thanks for creating this!

Reply:
Moved to `DataFormats/VertexReco` and renamed changing `Lepton` to `Track`